catalog_print.py

```diff
@@ -32,21 +28,39 @@ def get(self):
     session = self.request.get("session")
     if not session:
       logging.fatal("no session")
+    if not auth.HasPageAccess(institution, session, "materials"):
```

Comment:
Do we want the Catalog to be available to the student at every serving level?  Maybe remove lines 31-34?

If we later we decide we want to restrict access to only certain serving levels then we can always add something like the following code:

```python
    serving_levels = ("preregistration", "preferences", "schedule", "postregistration")
    if auth.ServingLevel(institution, session) not in serving_levels:
      auth.Redirect(institution, session)
      return
```